crates/ruff/src/rules/flake8_return/rules.rs

```diff
@@ -165,6 +165,53 @@ fn implicit_return_value(checker: &mut Checker, stack: &Stack) {
     }
 }
 
+const NORETURN_FUNCS: [&str; 8] = [
+    // stdlib
+    "os._exit",
+    "os.abort",
+    "posix.abort",
+    "sys.exit",
+    // third-party modules
+    "pytest.exit",
+    "pytest.fail",
+    "pytest.skip",
+    "pytest.xfail",
+];
```

Comment:
So we do have a system to support these -- grep around for `checker.resolve_call_path`. That will abstract away all the concerns around `import`, `import from`, aliasing, etc., and do the "right thing" in all cases to ensure that `func` is in fact a reference to the thing we care about here.

Reply:
`crates/ruff/src/rules/flake8_bugbear/rules/mutable_argument_default.rs` would be one concrete example of the syntax and usage.

Reply:
Wow. That made things... a lot simpler. But then, yes, that's what happens when a project already has the infrastructure in place to build upon. Thanks!